esm-samples/rollup/README.md

```diff
@@ -19,3 +19,65 @@ For additional information, see the [Build with ES modules](https://developers.a
 ## Commands
 
 For a list of all available `npm` commands see the scripts in `package.json`.
+
+## TypeScript
+This example can also be used with TypeScript sources. The following steps convert the example into a TypeScript starter.
+
+1. Rename src/main.js to src/main.ts
+2. Change npm packages
+	1. `npm uninstall -D @rollup/plugin-commonjs`
+	2. `npm install -D @rollup/plugin-typescript`
+	3. `npm install -D tslib`
+	4. `npm install -D typescript`
+3. Edit rollup.config.js
+	1. Change the line `import commonjs from "@rollup/plugin-commonjs";` to `import typescript from "@rollup/plugin-typescript";`
+	2. Change `input: "src/main.js",` to `input: "src/main.ts",`
+	3. Add `generatedCode: "es2015",` as an `output` parameter
+	4. Replace the `plugins` line `commonjs()` with `typescript()`
+4. Edit rollup.config.prod.js and make the same changes
+5. Add the file `tsconfig.json` to the folder containing `package.json`:
+```json
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowUnreachableCode": false,
+    "declaration": false,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "importsNotUsedAsValues": "preserve",
+    "lib": ["dom", "es2020"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": false,
+    "target": "es2020"
+  },
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
+}
+```
+6. Use `npm run build` and `npm run build:prod` as before
+
+Because TypeScript involves the extra step of transpilation, the watch feature doesn't appear to be possible, and so you might want to delete the following lines in rollup.config.js:
```

Comment:
`watch` is mentioned in the CHANGELOG, so I believe it is meant to work: https://github.com/rollup/plugins/blob/master/packages/typescript/CHANGELOG.md

Reply:
@dasa, thank you--I was wrong to include the deletion of `watch` et al. Works very nicely.